nih-sparc/sparc-app
# Events page - cu-qn19eu

# Description

The purpose of this PR is to add a landing page to show all events, and a featured event at the top.

## Type of change


# How Has This Been Tested?

- Go to the [news and events page](http://localhost:3000/news-and-events)
- Click on "Show All (5) Upcoming Events"
- You should be taken to the new Events page
- Clicking on "Upcoming" and "Past" tabs should change tabs
- Open `/pages/news-and-events/events/index.vue` and change line `143`  to `const featuredEvent = {}`. This should simulate that the featured event is not set.
- Go back to the new Events page
- The featured event should be removed, and clicking on "Show All (5) Upcoming Events" should show all upcoming events.


# Checklist:
